unlock-app/src/components/lock/ConfirmedKeyLock.js

```diff
@@ -28,7 +42,7 @@ ConfirmedKeyLock.propTypes = {
 
 export default ConfirmedKeyLock
 
-const Header = styled(LockHeader)`
-  background-color: var(--green);
-  color: var(--offwhite);
+const Body = styled(LockBody)`
+  border: 1px solid var(--green);
+  padding-top: 0;
```

Comment:
I think this `padding-top: 0;` is used on all the `Body` below? Maybe worth moving to `LockBody`?

Reply:
Yes, the only one that has a different value is `NoKeyLock`, so I will move the `padding-top: 0;` to `LockBody` and then override it for `NoKeyLock`.